specificatie/openapi.yaml

```diff
@@ -49,21 +49,45 @@ paths:
             example: "0226200000038820"
         - in: query
           name: postcode
-          description: Zoek WOZ-objecten met postcode
+          description: Zoek WOZ-objecten met postcode, in combinatie met huisnummer en eventueel met huisletter en/of huisnummertoevoeging
           required: false
           schema:
             type: string
             pattern: "^[1-9][0-9][0-9][0-9][A-Z][A-Z]$"
             example: "1234AB"
         - in: query
           name: huisnummer
-          description: Zoek WOZ-objecten met huisnummer"
+          description: Zoek WOZ-objecten met huisnummer, in combinatie met postcode en eventueel met huisletter en/of huisnummertoevoeging"
           required: false
           schema:
             type: integer
             minimum: 1
             maximum: 99999
-        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/fields"
+        - in: query
+          name: huisnummertoevoeging
+          description: |
+                        Zoek met een huisnummertoevoeging, in combinatie met postcode en huisnummer en eventueel huisnummtoevoeging.
+                        Een toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter. 
```

Comment:
Ik begrijp deze zin niet. Wat probeer je hiermee te zeggen?

Reply:
Uitleg over wat een huisnummertoevoeging is. Ik heb dit aangapast naar "Dit is eentoevoeging aan een huisnummer of een combinatie van huisnummer en huisletter."